transport/http/constants.go

```diff
@@ -28,6 +28,18 @@ var defaultConnTimeout = 500 * time.Millisecond
 
 // HTTP headers used in requests and responses to send YARPC metadata.
 const (
+	// ID for a request/response pair as chosen by the client. This corresponds
+	// to the Request.ID and Response.ID attribute.
+	IDHeader = "RPC-ID"
```

Comment:
Match case with other headers. Rpc-Id or Rpc-ID (there’s an argument that ID is not an initialism or acronym since it’s an abbreviation of Identifier, but like OK for okay, we pronounce it I.D., there’s no winning this game…)